templates/registration/profile.html

```diff
@@ -1,9 +1,14 @@
 {% extends "base.html" %}
 {% load humanize %}
+{% load staticfiles %}
```

Comment:
Small thing: I would prefer importing `staticfiles` before `humanize`. Can we change the order so that it looks like this?
```
{% load staticfiles %}
{% load humanize %}
```